cmd/mr_show.go

```diff
@@ -98,7 +106,43 @@ WebURL: %s
 		mr.Author.Username, milestone, labels, mr.WebURL)
 }
 
-func printMRDiscussions(discussions []*gitlab.Discussion) {
+func printMRDiscussions(discussions []*gitlab.Discussion, since string, mrNum int) {
+	NewAccessTime := time.Now().UTC()
+	viper.Reset()
+	mrEntry := fmt.Sprintf("mr%d", mrNum)
+	viper.SetConfigName("labmetadata")
+	viper.SetConfigType("hcl")
```

Comment:
As alluded to in #402 this doesn't need to be hcl, long term we want to move away from that for the other config as well. Go ahead and use whatever format you feel makes sense here.

Reply:
I'm going to stick with hcl for now just for the sake of consistency.   I'll invest some time in looking at other alternatives.